src/core/security/google_default_credentials.c

```diff
@@ -157,7 +174,7 @@ grpc_credentials *grpc_google_default_credentials_create(void) {
   if (result != NULL) goto end;
 
   /* Then the well-known file. */
-  result = create_jwt_creds_from_path(
+  result = create_refresh_token_creds_from_path(
```

Comment:
Just for this case (the well-known file) because that is what is populated by default by the gcloud SDK. I'll have a subsequent PR that will inspect the file and decide dynamically which type of credentials we have (and same thing for the environment variable).